sql/src/main/java/io/crate/operation/projectors/ProjectionToProjectorVisitor.java

```diff
@@ -306,13 +307,16 @@ public Projector visitSourceIndexWriterProjection(SourceIndexWriterProjection pr
         Input<?> sourceInput = ctx.add(projection.rawSource());
         Supplier<String> indexNameResolver =
             IndexNameResolver.create(projection.tableIdent(), projection.partitionIdent(), partitionedByInputs);
+        Settings tableSettings = TableSettingsResolver.get(clusterService.state(), projection.tableIdent(),
+            !projection.partitionedBySymbols().isEmpty());
         return new IndexWriterProjector(
             clusterService,
             nodeJobsCounter,
             threadPool.scheduler(),
             threadPool.executor(ThreadPool.Names.SEARCH),
             functions,
             clusterService.state().metaData().settings(),
```

Comment:
do we still need these settings here? They're used to get the `bulk.request_timeout`, which should be present in the partition-aware tableSettings as well?

Reply:
currently this settings is not a table but a config-only setting. maybe lets refactor this into a statement or table setting separately?

Reply:
Yep sounds good